components/organisms/calendar/calendar.tsx

```diff
@@ -3,28 +3,34 @@ import React, {useCallback, useEffect, useState} from "react";
 import * as S from "./calendar.styled";
 import {Date as DateComponent} from "./date/index";
 import {Header} from "./header/index";
+import {ExpandedTaskData} from "pages";
 import {Dayjs} from "utils/dayjs";
 
-export type Direction = "next" | "previous";
+type CalendarProps = {
+  pickedDate:string,
+  onDateClick: (date:string)=>void,
+  tasksByDate: Record<string, ExpandedTaskData[]>,
+}
 
-const WEEK_DAYS: string[] = ["S", "M", "T", "W", "T", "F", "S"];
+export type Direction = "next" | "previous";
+const WEEK_DAYS: string[] = ["sunday", "monday", "tuesday", "wednesday", "thursday", "friday", "saturday"];
```

Comment:
```suggestion
const WEEK_DAYS = ["sunday", "monday", "tuesday", "wednesday", "thursday", "friday", "saturday"];
```

string[] 을 빼면 좀더 자세한 타입이 됩니다.
(string[] 으로 타입 지정하면 WEEK_DAYS 는 말그대로 string 을 원소로 갖는 배열이고, string[] 으로 타입지정하지 않으면 ["sunday", "monday", "tuesday", "wednesday", "thursday", "friday", "saturday"] 그대로 기억됨)